scripts/rerun_edited_message_logs.py

```diff
@@ -12,7 +13,7 @@
 @app.command()
 def main(
     messages_path: str,
-    out_path: str | None = None,
+    out_path: Union[str, None] = None,
```

Comment:
I suppose I would point out, isnt this a bit redundant? `Optional[str]` by its self works fine but if you need the field set then theres two best practices here:

1. `Optional[str] = None # gives us a default value of None for the field, is polymorphic.`
2. `Optional[str] = Field(...) # gives us a default of None with an enforced str type validation - all classes should be dataclasses  anyway right?`